packages/matrix-identity-server/src/matrixDb/sql/pg.ts

```
@@ -63,17 +63,56 @@ class MatrixDBPg extends Pg<Collections> implements MatrixDBBackend {
             opts.host = RegExp.$1
             opts.port = parseInt(RegExp.$2)
           }
+          const sslType =
+            opts.ssl == null || opts.ssl === false
```

Biome (2.4.9)

[error] Using == may be unsafe if you are relying on type coercion.

(lint/suspicious/noDoubleEquals)